app/src/main/java/com/goryachok/forecastapp/view/recyclerviewadapters/HourlyForecastAdapter.kt

```diff
@@ -25,17 +24,17 @@ class HourlyForecastAdapter(private var forecast: List<Forecast>) :
         )
     }
 
-    override fun getItemCount(): Int = forecast.size
+    override fun getItemCount(): Int = forecast.size / 5
 
     override fun onBindViewHolder(holder: HourlyForecastViewHolder, position: Int) {
         val context = holder.itemView.context
         holder.date.text = forecast[position].dateText.substringAfter(" ")
         holder.desc.text = forecast[position].weather[0].description
```

Comment:
Please, use Collection.First() function instead of collection[0].

Reply:
Done